install/kubernetes/templates/_container_oci_hooks_setup.tpl

```diff
@@ -0,0 +1,19 @@
+{{- define "container.tetragon-oci-hook-setup" -}}
+- name: oci-hook-setup
+  securityContext:
+    privileged: true
```

Comment:
Could it reuse `{{- toYaml .Values.tetragon.securityContext | nindent 4 }}`? (it's `privileged: true` by default)

Reply:
I ended up defining a different context for the init container. Part of the reason we want the init container is that it needs certain permissions to install the hooks on the host. So I think it would be useful for it to have a different security context that the agent.